brvs-core/src/main/java/iroha/validation/transactions/startup/StartupLogic.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package iroha.validation.transactions.startup;
+
+/**
+ * Interface to add any logic being invoked on the service start
+ */
+public interface StartupLogic {
```

Comment:
Why you just don't use `@PostConstruct`?

Reply:
Just don't want to introduce huge responsibility classes and overload them with dependencies. I'd like to keep loosely coupled code. Moreover, some already existing functions could be refactored to this approach. Although there are many things nice to refactor..